scrapers/AdultEmpireCash.yml

```
@@ -0,0 +1,126 @@
+name: AdultEmpireCash
+# Sites ("Brands") from https://www.adultempirecash.com
+sceneByURL:
+  - action: scrapeXPath
+    scraper: sceneScraper
+    url:
+      - concoxxxion.com/
+      - elegantangel.com/
+      - filthykings.com/
+      - filthypov.com/
+      - forbiddenfruitsfilms.com/
+      - hornyhousehold.com/
+      - jayspov.net/
+      - jodiwest.com/
+      - kingsoffetish.com/
+      - latinoguysporn.com/
+      - lethalhardcore.com/
+      - manipulativemedia.com/
+      - mypervyfamily.com/
+      - pornstarstroker.com/ # aggregator site?
+      - reaganfoxx.com/
+      - severesexfilms.com/
+      - thirdworldxxx.com/
+      - touchmywife.com/
+      - westcoastproductions.com/
+
+  # VR Sites
+  - action: scrapeXPath
+    scraper: vrSceneScraper
+    url:
+      - lethalhardcorevr.com/
+      - whorecraftvr.com/
+
+movieByURL:
+  - action: scrapeXPath
+    scraper: movieScraper
+    url:
+      # - bangmytransass.com/ # Scenes published as movies?
+      # - blacktranswhackers.com/ # Scenes published as movies?
+      - concoxxxion.com/
+      - elegantangel.com/
```

```suggestion
      - elegantangel.com/
      - forbiddenfruitsfilms.com/
```
it seems that the movie scraper works also for forbiddenfruitfilms